config_defaults_inc.php

```diff
@@ -4140,6 +4140,8 @@
  * - DISPLAY_ERROR_INLINE  Display a one line error and continue execution.
  * - DISPLAY_ERROR_NONE    Suppress the error (no display). This is the default
  *                         behavior for unspecified errors constants.
+ * - DISPLAY_ERROR_HEADERS Stop and return an HTTP error header. No output
```

Comment:
Should we have a define or an API that when called at the beginning of a script page, it directs how the error is handled, without changing the valid values for this?  Having this option here means that users can use this globally.